stories/form/DatePicker.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { withInfo } from '@storybook/addon-info';
+import { DatePicker } from '@sparkpost/matchbox';
+import { DateUtils } from 'react-day-picker';
+
+export default {
+  title: 'Form|DatePicker',
+};
+
+const to = new Date();
+const from = new Date(new Date().setDate(new Date().getDate() - 15));
+const initial = new Date(new Date().setDate(new Date().getDate() - 30));
+
+const selectedDays = {
+  to,
+  from,
+};
+
+const modifiers = {
+  firstSelected: day => {
+    return DateUtils.isSameDay(day, selectedDays.from);
+  },
+  lastSelected: day => DateUtils.isSameDay(day, selectedDays.to),
+  inBetween: day => DateUtils.isDayBetween(day, selectedDays.from, selectedDays.to),
+};
+
+export const BasicDatepicker = withInfo({ propTables: [DatePicker] })(() => (
+  <DatePicker
+    modifiers={modifiers}
+    numberOfMonths={2}
```

Comment:
When changing this to more than 2 months the styles get a little wonky and it starts stacking months in a column. Is this intentional or should we limit the number of months allowed?

Reply:
Yeah.. maybe we should restrict this to either 1 or 2. Was thinking we would need 1 for smaller screens